src/components/Home.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react'
+import { getAllArticles } from '../utils/api-requests'
+import ArticleCards from './ArticleCards'
+
+function Home() {
+    const [articlesToBrowse, setArticlesToBrowse] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+    const [error, setError] = useState(null)
+
+useEffect(()=>{
+    setIsLoading(true)
+    getAllArticles().then(({articles}) => {      
+        setArticlesToBrowse(articles)
+        setIsLoading(false)
+        setError(null)
+    })
+    .catch((error) => {
+        setError(error)
+    })
+
+}, [])
+
+  return (
+    <>
```

Comment:
These fragments serve no purpose here because the component has a parent element (the `ul`)

Reply:
the `ul` should probably be part of the `ArticleCards` component